src/applications/gi-sandbox/containers/RefineYourSearch.jsx

```diff
@@ -92,7 +92,12 @@ export function RefineYourSearch({
 
   const updateResults = () => {
     if (search.tab === TABS.name) {
-      dispatchFetchSearchByNameResults(search.query.name, filters, version);
+      dispatchFetchSearchByNameResults(
+        search.query.name,
+        search.name.pagination.currentPage,
```

Comment:
Wouldn't we need to reset paging for new filtered results?

Reply:
You're right, we would